Sources/TabBar/View/TabBar.swift

```diff
@@ -112,7 +123,19 @@ extension TabBar {
             tabItemStyle : self.tabItemStyle,
             tabBarStyle  : style,
             selection    : self.selectedItem.$selection,
+            visibility   : self.visibility,
             content      : { self.content }
         )
     }
 }
+
+extension View {
+    @ViewBuilder public func visibility(_ visibility: TabBarVisibility) -> some View {
+        switch visibility {
+        case .visible:
+            self.transition(.move(edge: .bottom))
+        case .invisible:
+            hidden().transition(.move(edge: .bottom))
+        }
+    }
+}
```

Comment:
This is not good idea to make this function `public`, remove this keyword so it will have `internal` access level. Also, create `Extensions` folder and move this extension to a new file called `View.swift`